heron/packing/tests/java/com/twitter/heron/packing/AssertPacking.java

```diff
@@ -90,11 +90,10 @@ public static void assertNumInstances(Set<PackingPlan.ContainerPlan> containerPl
   public static void assertContainerRam(Set<PackingPlan.ContainerPlan> containerPlans,
                                         long maxRamforResources) {
     for (PackingPlan.ContainerPlan containerPlan : containerPlans) {
-      long containerRam = 0;
-      for (PackingPlan.InstancePlan instancePlan : containerPlan.getInstances()) {
-        containerRam += instancePlan.getResource().getRam();
-      }
-      Assert.assertTrue(containerRam <= maxRamforResources);
+      Assert.assertTrue("Container with id: " + containerPlan.getId() + " requires more RAM " + "("
```

Comment:
IMO it improves readability and is generally less typo-prone to have the text template, followed by the dynamic args that follow. It also makes it easier to update text without have so many `" + foo + " `s to work with. But this is subjective I suppose.